nixos/modules/services/networking/smokeping.nix

```diff
@@ -339,6 +339,10 @@ in
         mkdir -m 0755 -p ${smokepingHome}/cache ${smokepingHome}/data
         rm -f ${smokepingHome}/cropper
         ln -s ${cfg.package}/htdocs/cropper ${smokepingHome}/cropper
+        rm -f ${smokepingHome}/css
+        ln -s ${cfg.package}/htdocs/css ${smokepingHome}/css
+        rm -f ${smokepingHome}/js
+        ln -s ${cfg.package}/htdocs/js ${smokepingHome}/js
         rm -f ${smokepingHome}/smokeping.fcgi
         ln -s ${cgiHome} ${smokepingHome}/smokeping.fcgi
```

Comment:
ln has a -f parameter which makes the rm obsolete

Reply:
This is in the same style as the other commands in the surrounding context. @SuperSandro2000 feel free to open a followup PR to clean it up.

Reply:
#224996